cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1468,10 +1468,21 @@
   </property>
 
   <property>
+    <name>security.authorization.authorizer.extension.jar.path</name>
+    <value></value>
+    <description>
+      If an external authorization system is used for authorizing operations on CDAP entities, this property sets
+      the path to the bundled JAR file containing the extension source code. This property is only used when
+      authorization is enabled by setting security.authorization.enabled to true.
+    </description>
+  </property>
+
+  <property>
     <name>security.authorization.enabled</name>
     <value>false</value>
     <description>
-      Determines if authorization checks are enabled
+      Determines if authorization is enabled. When set to true, all operations in CDAP are authorized using the
+      authorizer implementation found at the property security.authorization.authorizer.extension.jar.path.
```

Comment:
We also have a dataset-backed implementation. However, we don't ship it with CDAP today. Created https://issues.cask.co/browse/CDAP-5296 to handle packaging/documentation for it.